src/components/FavorizeButton/FavorizeButton.js

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+FavorizeButton.propTypes = {
+  isFavorite: PropTypes.bool.isRequired,
+  onToggleFavorite: PropTypes.func.isRequired,
+  id: PropTypes.number.isRequired,
+}
+
+export default function FavorizeButton({ isFavorite, onToggleFavorite, id }) {
+  return (
+    <Button data-testid="favorize-button" onClick={() => onToggleFavorite(id)}>
```

Comment:
Why do you need the `data-testid`? Can you not find it by `getByRole('button')`?

Reply:
Also just saw, that you never use the test id. So you could just get rid of it!